C01/project-metadata/C01-ProjectId612-metadata.json

```diff
@@ -69,12 +69,44 @@
       }
     },
     "type": "Feature",
+    "place_name": "Taita–Taveta, Kenya",
     "geometry": {
       "type": "Point",
       "coordinates": [
         38.802647,
         -3.910022
       ]
     }
+  },
```

Comment:
Below are the marketing items that add enrichment to the Project Page. The `previewPhoto` is used for the "Project" card as seen in image above.

@S4mmyb I know we talked about keeping these in the metadata for now if we have them. Is that right?

Also, I don't believe it is crucial to add these to the VCS C01 standard (to keep things simple), but it is nice to have them as an option.

Reply:
Hey @mhagel, thanks for adding these! I agree they are nice to include, and for now we can do so without necessarily including them in the standards. The updates look good to me so far. Do we also want to include the benefits in the Impact section as well? Where are you pulling these from? 

Also it looks like a 'Biodiversity' and 'Avoided Deforestation' should be switched so the latter is the primary impact and the former (bio) is a co-benefit. 

![image](https://user-images.githubusercontent.com/67330492/163861649-d0d5f727-ac1a-41f6-b7ef-bfe45d85c1b8.png)

Reply:
@S4mmyb The Impacts and Co-benefit IRI's are in the Credit Class Version (from the postgres DB) metadata. We use these IRIs to fetch the images and text from the Sanity CMS.

Here's what that metadata currently looks like:

```
{
    "@type": "http://regen.network/CreditClass",
    "http://schema.org/url": {
        "@type": "http://schema.org/URL",
        "@value": "https://verra.org/wp-content/uploads/2019/09/VCS_Standard_v4.0.pdf"
    },
    "https://schema.org/url": "https://verra.org/wp-content/uploads/2019/09/VCS_Standard_v4.0.pdf",
    "http://regen.network/SDGs": {
        "@list": [
            {
                "@id": "http://regen.network/3GoodHealthAndWellBeing",
                "@type": "http://regen.network/SDG"
            },
            {
                "@id": "http://regen.network/4QualityEducation",
                "@type": "http://regen.network/SDG"
            },
            {
                "@id": "http://regen.network/8DecentWorkAndEconomicGrowth",
                "@type": "http://regen.network/SDG"
            },
            {
                "@id": "http://regen.network/12ResponsibleConsumptionAndProduction",
                "@type": "http://regen.network/SDG"
            },
            {
                "@id": "http://regen.network/13ClimateAction",
                "@type": "http://regen.network/SDG"
            },
            {
                "@id": "http://regen.network/15LifeOnLand",
                "@type": "http://regen.network/SDG"
            }
        ]
    },
    "http://regen.network/standard": {
        "http://schema.org/url": {
            "@type": "http://schema.org/URL",
            "@value": "https://verra.org/wp-content/uploads/2019/09/VCS_Standard_v4.0.pdf"
        },
        "http://schema.org/name": "VCS",
        "http://schema.org/version": "v4.0"
    },
    "http://regen.network/indicator": {
        "@id": "http://regen.network/AvoidedDeforestation",
        "@type": "http://regen.network/Indicator"
    },
    "http://regen.network/coBenefits": {
        "@list": [
            {
                "@id": "http://regen.network/Biodiversity",
                "@type": "http://regen.network/Indicator"
            },
            {
                "@id": "http://regen.network/Climate",
                "@type": "http://regen.network/Indicator"
            }
        ]
    },
    "http://regen.network/creditDenom": "VCU",
    "http://regen.network/certificateTitle": "Emission Reduction",
    "http://regen.network/offsetGenerationMethod": "Emission reduction"
}

Reply:
>Also it looks like a 'Biodiversity' and 'Avoided Deforestation' should be switched so the latter is the primary impact and the former (bio) is a co-benefit.

It looks like the UI places the "Primary Impact" label on the first item in the list. I will try to fix this in the code so that the primary is labeled correctly.

Reply:
Ahh that makes sense. Do we want to keep that in the Credit Class or migrate it to the projects as not all VCS projects might have these co-benefits? Also is there a need to align the credit class metadata in our postgresDB with the on-chain iri?

Reply:
Hmm, I guess we may need to move it to the project if the co-benefits differ. Although that feels like something that a credit-class defines? Suggesting we do this later, or make sure all the VCS co-benefits make sense.

Reply:
fixed